atomic_qt_design/qml/Exchange/OrderModal.qml

```diff
@@ -43,11 +47,9 @@ DefaultModal {
             Layout.alignment: Qt.AlignHCenter
             Layout.topMargin: 20
             font.pixelSize: Style.textSize3
-            visible: getStatus(details) !== status_swap_not_swap &&  // Is order
-                     (details.events !== undefined || // Has events, ongoing or
-                    details.am_i_maker === false) // Taker order with no events
-            color: visible ? getStatusColor(details) : ''
-            text_value: API.get().empty_string + (visible ? getStatusTextWithPrefix(details) : '')
+            visible: !details ? false : details.is_swap || !details.is_maker
+            color: !details ? "white" : visible ? getStatusColor(details.order_status) : ''
+            text_value: API.get().empty_string + (!details ? "" : visible ? getStatusTextWithPrefix(details.order_status) : '')
```

Comment:
This part is hard to read do to an excessive usage of the `ternary operator`, is it possible to make it more readeable ?

Reply:
I see. I'm trying to code keep property bindings not having functions if possible. `!details` part is just checking if the object is healthy. The rest of it is the real code. I can move them to the new line, that would help a lot.

Reply:
Moved real logic to the next line at property bindings.